app/src/main/java/com/stonks/ui/main/MainActivity.kt

```diff
@@ -37,7 +41,4 @@ class MainActivity : AppCompatActivity() {
 
             true
         }
-
-    // Variable storing currently selected fragment for displaying
-    private var selectedFragment : Fragment = DefaultFragment()  // TODO: place fragment corresponding to most left tab (now crypto)
```

Comment:
Why did you delete this and not replaced `DefaultFragment()` with `CurrencyFragment()`?

Reply:
It is not deleted just replaced to the top of the class